compare258.c

```diff
@@ -149,8 +151,10 @@ static inline uint32_t compare256_unaligned_64_static(const unsigned char *src0,
     uint32_t len = 0;
 
     do {
-        uint64_t sv = *(uint64_t *)src0;
-        uint64_t mv = *(uint64_t *)src1;
+        uint64_t sv;
+        uint64_t mv;
```

Comment:
As both variables have same type and they are not assigned a value that depend on each other, they can be declared on same line.

Reply:
Looks like this was resolved?